environments/.env.template

```diff
@@ -11,6 +11,7 @@ KEYCLOAK_FRONTEND_URL="${KEYCLOAK_FRONTEND_URL}"
 KEYCLOAK_DB_PASSWORD="${KEYCLOAK_DB_PASSWORD}"
 KEYCLOAK_READER_PASSWORD="${KEYCLOAK_READER_PASSWORD}"
 KEYCLOAK_UPLOADER_PASSWORD="${KEYCLOAK_UPLOADER_PASSWORD}"
+KEYCLOAK_REVIEWER_PASSWORD="${KEYCLOAK_REVIEWER_PASSWORD}"
 KEYCLOAK_DATALAND_ADMIN_PASSWORD="${KEYCLOAK_DATALAND_ADMIN_PASSWORD}"
 
```

Comment:
Why is "PGADMIN_PASSWORD" included in the section # Variables in relation with Backend ?
It has nothing to do with the backend.  It is only used in the pg admin service in the `docker-compose` stack.

Reply:
👍